README.rst

```diff
@@ -68,6 +70,71 @@ Requirements
 If using postgres as database driver, ensure the `postgres contrib`_ package is
 installed on server side.
 
+Recomended usage
+================
+
+Given target minion_id is 'srv'.
+
+/srv/salt/top.sls::
+
+    base:
+        'srv'
+            - saemref
+            - saemref.supervisor
+
+
+/srv/pillar/top.sls::
+
+    base:
+        'srv':
+            - saemref
+
+
+/srv/pillar/saemref.sls::
+
```

Comment:
What's the value of repeating this here, over pointing to `pillar.example`? It looks like repetition that would quickly get out of sync...